src/main/java/com/accantosystems/stratoss/vnfmdriver/model/alm/GenericExecutionRequestPropertyValue.java

```diff
@@ -0,0 +1,30 @@
+package com.accantosystems.stratoss.vnfmdriver.model.alm;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+/**
+ * Required as a general implemenation of DriverPropertyValue so the
+ * DriverPropertyDeserializer can deserialize into an instance for all non-key
+ * types
+ */
+@JsonInclude(value = JsonInclude.Include.NON_DEFAULT)
```

Comment:
Why the NON_DEFAULT annotation here - this caused issues for Brent (using a similar class in DTO) because it wasn't sending property types to the driver?